app/migrations/20240104144355-FlatInventory.cjs

```diff
@@ -0,0 +1,344 @@
+      await queryInterface.dropTable("SectorValue", { transaction });
+
+      /// EmissionsFactor ///
+      await renameColumns(
+        "EmissionsFactor",
+        {
+          emissions_factor_id: "id",
+          emissions_factor: "emissions_per_activity",
+          emissions_factor_url: "url",
+        },
+        queryInterface,
+        transaction,
+      );
+      await queryInterface.addColumn(
+        "EmissionsFactor",
+        "inventory_id",
+        Sequelize.UUID,
+        { transaction },
+      );
+      await queryInterface.addConstraint("EmissionsFactor", {
+        type: "foreign key",
+        name: "FK_EmissionsFactor.inventory_id",
+        fields: ["inventory_id"],
+        references: {
+          table: "Inventory",
+          field: "inventory_id",
+        },
+        onDelete: "CASCADE",
+        onUpdate: "CASCADE",
+        transaction,
+      });
+
+      /// GasToCO2Eq ///
+      await queryInterface.createTable(
+        "GasToCO2Eq",
+        {
+          gas: {
+            type: Sequelize.STRING(255),
+            primaryKey: true,
+            allowNull: false,
+          },
+          co2eq_per_kg: {
+            type: Sequelize.FLOAT,
+            allowNull: true,
+          },
+          co2eq_years: {
+            type: Sequelize.INTEGER,
+            allowNull: true,
+          },
+        },
+        { transaction },
+      );
+
+      /// GasValue ///
+      await queryInterface.createTable(
+        "GasValue",
+        {
+          id: {
+            type: Sequelize.UUID,
+            primaryKey: true,
+          },
+          inventory_value_id: {
+            type: Sequelize.UUID,
+            allowNull: true,
+            references: {
+              model: "InventoryValue",
+              key: "id",
+            },
+          },
+          emissions_factor_id: {
+            type: Sequelize.UUID,
+            allowNull: true,
+            references: {
+              model: "EmissionsFactor",
+              key: "id",
+            },
+          },
+          gas: { type: Sequelize.STRING, allowNull: true },
+          gas_amount: { type: Sequelize.BIGINT, allowNull: true },
+        },
+        { transaction },
+      );
+      await queryInterface.addConstraint("GasValue", {
+        type: "foreign key",
+        name: "FK_GasValue.inventory_value_id",
+        fields: ["inventory_value_id"],
+        references: {
+          table: "InventoryValue",
+          field: "id",
+        },
+        onDelete: "CASCADE",
+        onUpdate: "CASCADE",
+        transaction,
+      });
+      await queryInterface.addConstraint("GasValue", {
+        type: "foreign key",
+        name: "FK_GasValue.emissions_factor_id",
+        fields: ["emissions_factor_id"],
+        references: {
+          table: "EmissionsFactor",
+          field: "id",
+        },
+        onDelete: "SET NULL",
+        onUpdate: "SET NULL",
+        transaction,
+      });
+    });
+  },
+
+  async down(queryInterface) {
```

Comment:
Thanks for doing this. I think we're losing data, but that's probably OK.

Reply:
This is only meant to be used in a dev setup really. If we ever need to call it manually on the server we can adjust the down method to preserve the data beforehand.